modules/home/module.php

```diff
@@ -30,11 +30,15 @@ public function __construct() {
 
 		add_action( 'elementor/admin/menu/after_register', function ( Admin_Menu_Manager $admin_menu, array $hooks ) {
 			$hook_suffix = 'toplevel_page_elementor';
-			add_action( "admin_print_scripts-{$hook_suffix}", [ $this, 'enqueue_main_script' ] );
+			add_action( "admin_print_scripts-{$hook_suffix}", [ $this, 'enqueue_editor_scripts' ] );
```

Comment:
`enqueue_editor_scripts` - this is not an "editor" script.

Reply:
Updated.